darts/tests/models/forecasting/test_global_forecasting_models.py

```diff
@@ -274,32 +275,44 @@ def test_save_model_parameters(self, config):
                 output_chunk_length=3,
                 **tfm_kwargs,
             ),
+            LinearRegressionModel(lags=12),
         ],
     )
     def test_save_load_model(self, tmpdir_fn, model):
         # check if save and load methods work and if loaded model creates same forecasts as original model
         model_path_str = type(model).__name__
+        model_clean_path_str = type(model).__name__ + "_clean"
+
         full_model_path_str = os.path.join(tmpdir_fn, model_path_str)
+        full_model_clean_path_str = os.path.join(tmpdir_fn, model_clean_path_str)
 
         model.fit(self.ts_pass_train)
         model_prediction = model.predict(self.forecasting_horizon)
 
         # test save
         model.save()
-        model.save(model_path_str)
+        model.save(full_model_path_str)
 
-        assert os.path.exists(full_model_path_str)
-        assert (
-            len([
-                p for p in os.listdir(tmpdir_fn) if p.startswith(type(model).__name__)
-            ])
-            == 4
-        )
+        temp_training_series = model.training_series
+        temp_future_cov = model.future_covariate_series
+        temp_past_cov = model.past_covariate_series
+        model.save(full_model_clean_path_str, clean=True)
+        # No side effect to drop the training series
+        assert temp_training_series == model.training_series
+        assert temp_future_cov == model.future_covariate_series
+        assert temp_past_cov == model.past_covariate_series
 
         # test load
-        loaded_model = type(model).load(model_path_str)
+        loaded_model = type(model).load(full_model_path_str)
+        loaded_model_drop_str = type(model).load(full_model_clean_path_str)
 
         assert model_prediction == loaded_model.predict(self.forecasting_horizon)
+        assert model_prediction == loaded_model_drop_str.predict(
+            self.forecasting_horizon, self.ts_pass_train
+        )
+
+        # test save with clean
+        assert loaded_model_drop_str.training_series is None
```

Comment:
Could you add a check that the loaded clean model raises an error when calling `predict()` without passing a `series`?